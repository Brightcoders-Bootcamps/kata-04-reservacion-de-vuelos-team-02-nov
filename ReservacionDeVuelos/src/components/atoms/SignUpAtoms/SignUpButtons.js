import React from 'react';
import {View, Text, TouchableOpacity, Image} from 'react-native';
import {styles} from '../../../styles/SignUpStyles';
import GoogleIcon from '../../../assets/images/google.png';
import {SignUpConstants} from '../../../utils/Constants';
const SignUpButtons = () => {
  return (
    <View style={styles.btnSignupContainer}>
      <TouchableOpacity style={styles.btnSignup}>
        <Text style={styles.btnSignupText}>{SignUpConstants.SignUpButton}</Text>
      </TouchableOpacity>
      <Text style={styles.or}>or</Text>
      <TouchableOpacity style={styles.btnSignup}>
        <Image
          source={GoogleIcon}
          style={[styles.googleIcon, {marginLeft: -75}]}
        />
        <Text style={[styles.btnSignupText, {marginLeft: 50}]}>
          {SignUpConstants.GoogleSignUpButton}
        </Text>
      </TouchableOpacity>
      <View style={styles.loginTextContainer}>
        <Text>{SignUpConstants.HasAnAccount}</Text>
        <TouchableOpacity>
          <Text style={styles.LogInText}>{SignUpConstants.LogIn}</Text>
        </TouchableOpacity>
      </View>
    </View>
  );
};

export default SignUpButtons;
