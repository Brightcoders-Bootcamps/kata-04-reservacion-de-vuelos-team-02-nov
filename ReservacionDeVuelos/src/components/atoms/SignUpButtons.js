import React from 'react';
import {View, Text, Button, TouchableOpacity, Image} from 'react-native';
import {styles} from '../../styles/SignUpStyles';
import GoogleIcon from '../../assets/images/google.png';
const SignUpButtons = () => {
  return (
    <View style={styles.btnSignupContainer}>
      <TouchableOpacity style={styles.btnSignup}>
        <Text style={styles.btnSignupText}>Sign Up</Text>
      </TouchableOpacity>
      <Text style={styles.or}>or</Text>
      <TouchableOpacity style={styles.btnSignup}>
        <Image
          source={GoogleIcon}
          style={[styles.googleIcon, {marginLeft: -75}]}
        />
        <Text style={[styles.btnSignupText, {marginLeft: 50}]}>
          Sign Up with Google
        </Text>
      </TouchableOpacity>
      <Text style={styles.loginTextContainer}>
        Already have an account?{' '}
        <TouchableOpacity>
          <Text style={styles.LogInText}> Log In</Text>
        </TouchableOpacity>
      </Text>
    </View>
  );
};

export default SignUpButtons;
