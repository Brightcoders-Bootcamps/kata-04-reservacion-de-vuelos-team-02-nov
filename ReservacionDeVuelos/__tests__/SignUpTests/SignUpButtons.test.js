import React from 'react';
import SignUpButtons from '../../src/components/atoms/SignUpAtoms/SignUpButtons';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUpButtons />).toJSON();
  expect(tree).toMatchSnapshot();
});
