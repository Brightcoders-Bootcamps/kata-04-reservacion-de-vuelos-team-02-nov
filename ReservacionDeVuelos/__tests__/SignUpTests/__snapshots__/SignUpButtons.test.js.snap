// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 15,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#b6b7ba",
        "borderRadius": 10,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 5,
        "opacity": 1,
        "padding": 12,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 18,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Sign Up
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#b6b7ba",
        "marginBottom": 10,
        "marginTop": 10,
        "textAlign": "center",
      }
    }
  >
    or
  </Text>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#b6b7ba",
        "borderRadius": 10,
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 5,
        "opacity": 1,
        "padding": 12,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/images/google.png",
        }
      }
      style={
        Array [
          Object {
            "height": 20,
            "marginTop": 3,
            "width": 20,
          },
          Object {
            "marginLeft": -75,
          },
        ]
      }
    />
    <Text
      style={
        Array [
          Object {
            "color": "#FFF",
            "fontSize": 18,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          Object {
            "marginLeft": 50,
          },
        ]
      }
    >
      Sign Up with Google
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "marginTop": 10,
        "textAlign": "center",
      }
    }
  >
    <Text>
      Already have an account?
    </Text>
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#2D02FF",
          }
        }
      >
        Log In
      </Text>
    </View>
  </View>
</View>
`;
